packages/conf-gcc/conf-gcc.1.0/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "2.0"
+maintainer: "unixjunkie@sdf.org"
+authors: ["Francois Berenger" "Francois Bobot"]
+homepage: "https://github.com/ocaml/opam-repository"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+license: "GPL-2+"
+build: [["which" "gcc"]]
```

Comment:
I think it would be simpler to just call `gcc --version` instead of relying on `which`